packages/mwp-config/stylelint/stylelint-config-default.js

```diff
@@ -0,0 +1,206 @@
+		'declaration-block-semicolon-newline-before': 'never-multi-line',
+		'declaration-block-semicolon-space-after': 'never',
+		'declaration-block-semicolon-space-before': 'never',
+		'declaration-block-single-line-max-declarations': 1,
+		'declaration-block-trailing-semicolon': 'always',
+		'declaration-colon-space-after': 'always',
+		'declaration-colon-space-before': 'never',
+		'declaration-empty-line-before': 'never',
+		'declaration-property-unit-whitelist': {
+			'line-height': [],
+			'/^animation/': ['ms'],
+			'/^transition/': ['ms'],
+		},
+		'declaration-property-value-blacklist': {
+			outline: ['outline'],
+			// no font-size values under 12px or .75rem
+			'font-size': [
+				'/^(((0?[.](7[5-9]|[8-9][0-9])|[1-9].?([0-9]?[0-9]?[0-9])*)rem)|(?:(?:1[2-9]|[2-9][0-9]|[1-9][0-9][0-9])px))$/',
+			],
+		},
+		'font-family-name-quotes': 'always-where-recommended',
+		'font-family-no-duplicate-names': true,
+		'font-family-no-missing-generic-family-keyword': true,
+		'font-weight-notation': 'numeric',
+		'function-calc-no-unspaced-operator': true,
+		'function-comma-newline-after': 'never-multi-line',
+		'function-comma-newline-before': 'never-multi-line',
+		'function-comma-space-after': 'always',
+		'function-comma-space-before': 'never',
+		'function-linear-gradient-no-nonstandard-direction': true,
+		'function-max-empty-lines': 0,
+		'function-name-case': 'lower',
+		'function-parentheses-newline-inside': 'never-multi-line',
+		'function-parentheses-space-inside': 'never',
+		'function-url-quotes': 'always',
+		'function-url-scheme-blacklist': ['/^data:image/'],
+		'function-whitespace-after': 'always',
+		indentation: 'tab',
+		'length-zero-no-unit': true,
+		'max-nesting-depth': 3,
+		'media-feature-colon-space-after': 'always',
+		'media-feature-colon-space-before': 'never',
+		'media-feature-name-case': 'lower',
+		'media-feature-name-no-unknown': true,
+		'media-feature-parentheses-space-inside': 'never',
+		'media-feature-range-operator-space-after': 'always',
+		'media-feature-range-operator-space-before': 'always',
+		'media-query-list-comma-space-after': 'always',
+		'media-query-list-comma-space-before': 'never',
+		'no-duplicate-at-import-rules': true,
+		'no-duplicate-selectors': true,
+		'no-empty-source': true,
+		'no-eol-whitespace': true,
+		'no-extra-semicolons': true,
+		'no-unknown-animations': true,
+		'number-leading-zero': 'always',
+		'number-max-precision': 2,
+		'number-no-trailing-zeros': true,
+		'property-blacklist': 'text-rendering',
+		'property-case': 'lower',
+		'property-no-unknown': true,
+		'property-no-vendor-prefix': true,
+		'rule-empty-line-before': 'always',
+		'selector-attribute-brackets-space-inside': 'never',
+		'selector-attribute-operator-blacklist': '|=',
+		'selector-attribute-operator-space-after': 'never',
+		'selector-attribute-operator-space-before': 'never',
+		'selector-attribute-quotes': 'always',
+		'selector-class-pattern': [
```

Comment:
Do we need to set a level for this?

Reply:
Alternatively, we could pull this out of shared rules and export a separate, stricter set for use in SDS projects:

```
const SDSFrameworkConfig { rules: { 'selector-class-pattern': ... } }
```

Reply:
Severity level:
Going to come back to this after I fix all error/warnings in MWC

Only including in SDS projects:
Totally makes sense